ckcp/hack/util/util-development-patch.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 The Pipeline Service Authors.
```

Comment:
Perhaps rename to file to something more appropriate - update-git-reference.sh

Reply:
sure.